src/server/services/cache.service.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Provides a class for temporary data storage (caching)
+ * @module services/cache
+ */
+
+const NodeCache = require("node-cache");
+const { logEmitter } = require("./logging.service");
+
+// Enforce one variable per Cache object
+const key = "x";
+
+class Cache {
```

Comment:
A style point here...this is the first class we've used across the service (functional programming being the preferred paradigm). Up to you guys, but is it worth retaining that consistency?